mkdocs-nav.yml

```diff
@@ -174,6 +175,7 @@ nav:
       - "api/color.md"
       - "api/command.md"
       - "api/containers.md"
+      - "api/content_switcher.md"
```

Comment:
Given that `ContentSwitcher` is within the widgets, I'm wondering why there is an `api/content_switcher.md` at all.